2200095/problom3.py

```diff
@@ -0,0 +1,14 @@
+# 10505. 소득 불균형
+T = int(input())
+cnt = 1
+for t in range(T):
+    test_case = int(input())
+    input_list = list(map(int,input().split()))
+    res_list = []
+    for i in input_list:
+        avg = sum(input_list)/len(input_list)
+        if i <= avg:
+            res_list.append(i)
```

Comment:
```suggestion
    avg = sum(input_list)/len(input_list)
    for i in input_list:
        if i <= avg:
            res_list.append(i)
```
`avg`를 `for` 문 밖에서 계산하는 방법도 있습니다!